### PR TITLE
Add docs about the Hosts metrics tab to serverless

### PR DESCRIPTION
**Pull request by dedemorton**
Ports #3939 to serverless.

Note that there is some content currently missing from the serverless docs. I don't have time to port that content today so I've created an issue here: https://github.com/elastic/observability-docs/issues/3954

Preview is here: https://docs-elastic-odt7mapih-elastic-dev.vercel.app/serverless/observability/analyze-hosts

note that the base URL changes when new commits are pushed.

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _co-preview-docs_ successfully!

* https://docs-elastic-p31ip9w4f-elastic-dev.vercel.app

* Built with commit 85194d588c022db5b5f2a451b1f859d36635411e

> Issues? Visit #next-docs in Slack

File: docs/en/serverless/transclusion/host-details.mdx [hunk 1]
```diff
@@ -2,34 +2,43 @@
 
 <DocAccordion buttonContent="Overview">
 
-![Host metrics](../images/metrics-overlay.png)
-
-The **Overview** tab displays metrics about the selected host, including CPU usage,
-normalized load, memory usage, disk usage, network traffic, and the log rate.
+The *Overview* tab displays key metrics about the selected host, such as CPU usage,
```
Comment (dedemorton) [inline docs/en/serverless/transclusion/host-details.mdx:5]:
```suggestion
The **Overview** tab displays key metrics about the selected host, such as CPU usage,
```
```suggestion
The *Overview* tab displays key metrics about the selected host, such as CPU usage,
```

Comment (dedemorton) [inline docs/en/serverless/transclusion/host-details.mdx:5]:
```suggestion
The **Overview** tab displays key metrics about the selected host, such as CPU usage,
```

File: docs/en/serverless/transclusion/host-details.mdx [hunk 2]
```diff
@@ -2,34 +2,43 @@
...
 
+![Host metadata](../images/metadata-overlay.png)
+
 </DocAccordion>
 
-<DocAccordion buttonContent="Processes">
+<DocAccordion buttonContent="Metrics">
 
-![Host processes](../images/processes-overlay.png)
+The *Metrics* tab shows host metrics organized by type and is more complete than the view available in the *Overview* tab.
```
Comment (dedemorton) [inline docs/en/serverless/transclusion/host-details.mdx:35]:
```suggestion
The **Metrics** tab shows host metrics organized by type and is more complete than the view available in the *Overview* tab.
```
```suggestion
The *Metrics* tab shows host metrics organized by type and is more complete than the view available in the *Overview* tab.
```

**Review by mdbirnstiehl — APPROVED** [review]
LGTM!

**Pull request merged**